TreeSizeDuplicates.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Dec 11 11:43:25 2019
+This program will read a file in from TreeSize and determine what duplicates
+exist and mark them "Duplicate" or "Different" in a new column.
+Program inserts a new column before column A.
+Comparison is of adjacent rows using columns B, D, E, and F.
+Sort by column A prior to running is currently required.
+@author: ad414d
+"""
+
+#Import function to read xlsx file.
+from openpyxl import load_workbook
+
+#Define variable for the TreeSize workbook.
+wb=load_workbook('C:\\aaTanker\TreeSize\jrl me folder - duplicate.xlsx')

```

Comment:
All backslashes should be doubled here. A backslash in a string is what's known as an escape character; for example, \n is an escape sequence representing a newline. The reason this works as-is is because \T and \j aren't valid escape sequences so it guesses what you mean, and it happens to guess correctly.

Alternatively, use raw strings. A raw string is one that has the letter r before the opening quote, and escape sequences are ignored. So you could say here
`wb=load_workbook(r'C:\aaTanker\TreeSize\jrl me folder - duplicate.xlsx')`

Reply:
I prefer the shorter version of raw strings and added the "r" in front.

Reply:
So do I. The only advantage to doubling up on backslashes is that it's more universal across languages (each language does raw strings differently if they do at all).